### PR TITLE
Removes warnings by dmd 2.084.0.

### PR DESCRIPTION
**Pull request by kntroh**
I removed warnings by dmd 2.084.0.

### CODE REVIEW

File: org.eclipse.swt.snippets/src/org/eclipse/swt/snippets/Snippet144.d [hunk 1]
```diff
@@ -109,7 +109,7 @@ void main() {
                 auto t = elapsed.stop() * 1_000;
             } else { // Phobos
                 elapsed.stop();
-                auto t = elapsed.peek.msecs;
+                auto t = elapsed.peek().total!"msecs"();
```
Comment (jacob-carlborg) [inline org.eclipse.swt.snippets/src/org/eclipse/swt/snippets/Snippet144.d]: Why this change?

Comment (kntroh) [reply org.eclipse.swt.snippets/src/org/eclipse/swt/snippets/Snippet144.d]: The peek() of deprecated `std.datetime.StopWatch` returns TickDuration, However peek() of `std.datetime.stopwatch.StopWatch` returns `Duration` (`TickDuration` is deprecated). To take a millisecond value from `Duration` we need to use the total().

Comment (jacob-carlborg) [reply org.eclipse.swt.snippets/src/org/eclipse/swt/snippets/Snippet144.d]: Ok, I see.

**Comment by jacob-carlborg** [comment]
Thanks.
